ext/cl_khr_external_memory.asciidoc

```diff
@@ -181,10 +182,15 @@ Following new enums are added to the list of supported _param_names_ by {clGetDe
 | {CL_DEVICE_EXTERNAL_MEMORY_IMPORT_HANDLE_TYPES_KHR}
   | {cl_external_memory_handle_type_khr_TYPE}[]
       | Returns the list of importable external memory handle types supported by _device_.
+| {CL_DEVICE_EXTERNAL_MEMORY_IMPORT_ASSUME_LINEAR_HANDLE_TYPES_KHR}
```

Comment:
Should the name explicitly mention images and/or tiling? Something like `CL_DEVICE_EXTERNAL_MEMORY_IMPORT_ASSUME_LINEARLY_TILED_IMAGES_HANDLE_TYPES_KHR` would capture it all but gets a bit long-winded. Any better proposals?

Reply:
Compromise?  `CL_DEVICE_EXTERNAL_MEMORY_IMPORT_ASSUME_LINEAR_IMAGE_HANDLE_TYPES_KHR`